### ISSUE TITLE
[tailwindcss] [ERROR] PathNotFoundException: Cannot open file, path = '/private/var/folders/

### ISSUE THREAD

**Issue by JAICHANGPARK** [schultek/jaspr#350]
**Description**

**Steps To Reproduce**

1. install tailwindcss https://docs.page/schultek/jaspr/eco/tailwind this instruction 
2. Create a new project with `jaspr create  my_app` & setup client 
3. and setup this readme instruction https://github.com/schultek/jaspr/tree/main/packages/jaspr_tailwind 
4. `jaspr serve`

**Expected Behavior**

**Additional Context**

```
name: my_web_client
description: A new jaspr project.
version: 0.0.1

environment:
  sdk: '>=3.0.0 <4.0.0'

dependencies:
  jaspr: ^0.16.2
  jaspr_router: ^0.6.0

dev_dependencies:
  build_runner: ^2.4.0
  jaspr_web_compilers: ^4.0.10+1
  jaspr_builder: ^0.16.2
  jaspr_lints: ^0.1.2
  lints: ^3.0.0
  jaspr_tailwind: ^0.3.0

jaspr:
  mode: client

```

```
 jaspr serve
[CLI] Running jaspr in client rendering mode.
[BUILDER] [ERROR] PathNotFoundException: Cannot open file, path = '/private/var/folders/5f/5cdjr19d76sf4j511p_vt90r0000gn/T/scratch_spaceuX)
[BUILDER] [ERROR] Failed after 234ms
[CLI] [ERROR] Failed building web assets. There is probably more output above.
[CLI] [ERROR] Could not start dev server due to build errors.
[CLI] Shutting down...
[CLI] Terminating web compiler...
```

**Comment by JAICHANGPARK** [comment]
tailwind.config.js

```
/** @type {import('tailwindcss').Config} */
module.exports = {
    content: ['./{lib,web}/**/*.dart'],
    theme: {
      extend: {},
    },
    plugins: [
        require('daisyui'),
    ],
  }
```

**Comment by schultek** [comment]
Maybe this is the same as #289 and fixed in `jaspr_tailwind v0.3.1`.

Please test with the new version and reopen if the problem is still there.

**Comment by achmadfeby** [comment]
hi, i have the same problem, i already update to jaspr_tailwind v0.3.3 and is still not working
